bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -34,3 +43,6 @@ tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 # keep in sync with uniffi dependency in matrix-sdk-crypto-ffi, and uniffi_bindgen in ffi CI job
 uniffi = { git = "https://github.com/mozilla/uniffi-rs", rev = "6d6724a26b119032588c5b3d59ac4d09f902e778" }
 uniffi_macros = { git = "https://github.com/mozilla/uniffi-rs", rev = "6d6724a26b119032588c5b3d59ac4d09f902e778" }
+futures-signals = { version = "0.3.28", optional = true }
+ruma = { version = "0.6.2", optional = true }
```

Comment:
The rest of the crate is going through `matrix_sdk::ruma`. I think it should either be an unconditional dependency and be used consistently, or be removed as a dependency.